.bazelci/tests.yml

```diff
@@ -66,7 +66,7 @@ ubuntu1804: &ubuntu
     - "//distro:relnotes"
     - "//doc_build:*"
 
-centos7: &centos
+centos7_java11_devtoolset10: &centos
```

Comment:
Tests are still failing with the C++ std error.
Should the change be to the `platform`?

Reply:
You are absolute correct!